spec/factories/loans.rb

```diff
@@ -55,11 +55,12 @@
 
 
     trait :active do
-      status_option_id Loan::STATUS_OPTIONS.value_for('Active')
+      # note: can't seem to count on the seed.db data having been loaded when these are executed
+      status_option_id 1 #Loan.status_option_set.value_for_label('Active')
```

Comment:
Thanks for taking the time to get Tom's input.  Definitely important to move forward, so considered settled.  I'll sort out how to work most effectively within these constraints.  One more point to clarify.  It sounds like, in addition to using a generic hook which always guaranteed the availability of the seed data, that invoking the seed_data on an ad hoc basis just for the tests which needed it also considered a bad practice?

Is the Elmo codebase something which is open source or some that I can look at for more reference?  The code example around the `get_loan_status` that provided isn't quite relevant.  The challenge with the OptionSets is that they are essentially singleton objects (with dependent data) associated with the class.  But maybe something like this:
```
def ensure_loan_status_option_set
  loan_status = OptionSet.find_or_create_by(division: Division.root, model_type: Loan.name, model_attribute: 'status')
  loan_status.options.find_or_create_by(value: Loan::STATUS_ACTIVE_VALUE).set_label_list(en: 'Active', es: 'Prestamo Activo')
end
```
And put that in a `before` block for the various sections of code which depend on this configuration data.  This is essentially putting little pieces of the seeds.rb into helper methods which can be invoked when needed for very specific pieces of test code or factory setters.  Is that sufficiently capturing the spirit of your suggestion?